plugins/woo-ai/src/utils/productData.ts

```diff
@@ -1,32 +1,25 @@
 /**
  * Internal dependencies
  */
-import { Attribute, ProductData } from './types';
-import { getTinyContent, getPostId } from '.';
+import { Attribute } from './types';
```

Comment:
Let's also remove the type definition from the ./types.

Reply:
Not sure I understand. We're still using this type, so I'm not sure why we want to delete it?

Reply:
Sorry I meant the ProductData type

Reply:
Right! Thanks 👍🏻